build.gradle

```diff
@@ -274,7 +274,11 @@ configure modsRegular,
             forkEvery = 32
         }
 
-        systemProperty 'Configuration.rootFile', 'dh-tests.prop'
+        if (findProperty('shortTests') == 'true') {
+            systemProperty 'Configuration.rootFile', 'dh-short-tests.prop'
+        } else {
+            systemProperty 'Configuration.rootFile', 'dh-tests.prop'
+        }
```

Comment:
I think this could be equivalently accomplished by:

```gradle
if (findProperty('shortTests') == 'true') {
    systemProperty 'TstUtils.shortTests', 'true'
}
```
With the added advantage that it saves us from duplicating the test file. If we think there is further value in shortTests changing configuration values, we should instead create `dh-test-common.prop` and have `dh-tests.prop` and `dh-short-tests.prop` extend from the common one.